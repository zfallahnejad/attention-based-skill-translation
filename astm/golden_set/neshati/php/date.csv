250259,date
67332,date
118068,date
476,date
1438393,date
187606,date
507674,date
3859027,date
324584,date
1338292,date
862594,date
208809,date
1144035,date
762073,date
138475,date
50079,date
87015,date
541091,date
1570534,date
251311,date
1491895,date
889949,date
13508,date
350858,date
3933332,date
4833,date
204701,date
421223,date
2253331,date
696364,date
502381,date
2364629,date
871050,date
634824,date
417562,date
