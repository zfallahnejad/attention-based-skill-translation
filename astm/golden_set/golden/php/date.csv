250259,172,155,0.5259938837920489
67332,62,77,0.4460431654676259
118068,46,64,0.41818181818181815
476,33,43,0.4342105263157895
1438393,31,20,0.6078431372549019
187606,23,15,0.6052631578947368
507674,22,18,0.55
3859027,21,8,0.7241379310344828
324584,20,23,0.46511627906976744
1338292,17,16,0.5151515151515151
862594,15,7,0.6818181818181818
208809,15,17,0.46875
762073,14,17,0.45161290322580644
1144035,14,9,0.6086956521739131
50079,12,17,0.41379310344827586
138475,12,13,0.48
87015,12,14,0.46153846153846156
1570534,11,10,0.5238095238095238
541091,11,6,0.6470588235294118
251311,10,7,0.5882352941176471
1491895,10,12,0.45454545454545453
889949,10,8,0.5555555555555556
350858,9,4,0.6923076923076923
13508,9,13,0.4090909090909091
3933332,9,9,0.5
204701,8,8,0.5
421223,8,6,0.5714285714285714
2253331,8,8,0.5
4833,8,5,0.6153846153846154
696364,8,4,0.6666666666666666
634824,7,8,0.4666666666666667
417562,7,10,0.4117647058823529
2364629,7,6,0.5384615384615384
871050,7,9,0.4375
502381,7,3,0.7
